src/System.IO.FileSystem/tests/Base/FileGetSetAttributes.cs

```diff
@@ -33,6 +40,11 @@ public void SettingAttributes_Unix(FileAttributes attributes)
         public void SettingAttributes_Windows(FileAttributes attributes)
         {
             string path = CreateItem();
+            AssertSettingAttributes(path, attributes);
+        }
+
+        private void AssertSettingAttributes(string path, FileAttributes attributes)
+        {
             SetAttributes(path, attributes);
             Assert.Equal(attributes, GetAttributes(path));
```

Comment:
Is it not possible the file could have additional attributes beyond the ones the test is explicitly setting?  Seems like this should check that it has at least the required attributes, rather than exactly the required attributes?  Or is there something else that's guaranteeing the equality?

Reply:
This was just extracted from multiple methods. Is it possible that the temp file that tests create will have some attributes already?

Reply:
> Is it possible that the temp file that tests create will have some attributes already?

Not sure.  I guess we can leave it as is, and if the test ever fails, it'll be pretty obvious why.